### DISCUSSION TITLE
Grafana and Prometheus storage allocation

### DISCUSSION THREAD

**Discussion by ilanKushnir** [carlosedp/cluster-monitoring#143] [Q&A]
in the vars file I saw that you allocated the PVs sizes as follows:
```
    prometheusSizePV: '2Gi',
    grafanaSizePV: '20Gi',
```
https://github.com/carlosedp/cluster-monitoring/blob/master/vars.jsonnet#L77

shouldn't Prometheus get the bigger storage allocation? because it is scraping and holding all the collected metrics, Grafana should just display it right?

in my current setup, I deployed the services 4 days ago and this is the current storage usage:
* Prometheus is using `17.5 Gi`
* Grafana is using `160 Mi`

am I right or am I missing something here?
thanks!

and huge thanks for the awesome configuration!!!

**Comment by carlosedp** [comment]
See my answer at https://github.com/carlosedp/cluster-monitoring/discussions/147
